packages/react-scripts/package.json

```diff
@@ -28,59 +28,59 @@
   },
   "types": "./lib/react-app.d.ts",
   "dependencies": {
-    "@babel/core": "^7.14.2",
+    "@babel/core": "^7.15.8",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.1",
     "@svgr/webpack": "^5.5.0",
-    "babel-jest": "^27.1.0",
+    "babel-jest": "^27.2.5",
     "babel-loader": "^8.2.2",
     "babel-plugin-named-asset-import": "^0.3.7",
     "babel-preset-react-app": "^10.0.0",
     "bfj": "^7.0.2",
-    "browserslist": "^4.16.5",
+    "browserslist": "^4.17.3",
     "camelcase": "^6.2.0",
     "case-sensitive-paths-webpack-plugin": "^2.4.0",
-    "css-loader": "^6.2.0",
-    "css-minimizer-webpack-plugin": "^3.0.2",
-    "dotenv": "^9.0.2",
+    "css-loader": "^6.4.0",
+    "css-minimizer-webpack-plugin": "^3.1.1",
+    "dotenv": "^10.0.0",
     "dotenv-expand": "^5.1.0",
-    "eslint": "^7.30.0",
+    "eslint": "^7.32.0",
     "eslint-config-react-app": "^6.0.0",
-    "eslint-webpack-plugin": "^2.5.4",
+    "eslint-webpack-plugin": "^3.0.1",
     "file-loader": "^6.2.0",
     "fs-extra": "^10.0.0",
     "html-webpack-plugin": "^5.3.2",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.1.0",
-    "jest-resolve": "^27.1.0",
-    "jest-watch-typeahead": "^0.6.4",
-    "mini-css-extract-plugin": "^2.1.0",
-    "postcss": "^8.3.5",
+    "jest": "^27.2.5",
+    "jest-resolve": "^27.2.5",
+    "jest-watch-typeahead": "^1.0.0",
+    "mini-css-extract-plugin": "^2.4.2",
+    "postcss": "^8.3.9",
     "postcss-flexbugs-fixes": "^5.0.2",
     "postcss-loader": "^6.1.1",
     "postcss-normalize": "^10.0.1",
     "postcss-preset-env": "^6.7.0",
-    "prompts": "^2.4.1",
+    "prompts": "^2.4.2",
     "react-app-polyfill": "^2.0.0",
     "react-dev-utils": "^11.0.4",
```

Comment:
I think `react-dev-utils` should be bumped up.
https://github.com/facebook/create-react-app/issues/11567